details.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
+
+@view_function
+def process_request(request):
+    prod = cmod.Product.objects.get(id = request.dmp.urlparams[0])
+    print(prod)
+
+    if prod in request.last_five:
+        request.last_five.remove(prod)
+
+    request.last_five.insert(0,prod)
+
+    if len(request.last_five) > 6:
+        request.last_five.pop(-1)
+
+    context = {
+        "prod" : prod,
+
+    }
+    return request.dmp.render('details.html', context)
+# def process_request(request, cat:cmod.Category=None, pnum:int=1):
```

Comment:
If you aren't going to use all of this code, it might be a good idea to delete it to increase

Reply:
yeah i better go through and delete the unused stuff.